Storage/SQL/ReadingListSchema.swift

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import XCGLogger
+
+let TableReadingListItems = "items"
+
+private let AllTables: [String] = [
+    TableReadingListItems
+]
+
+private let log = Logger.storageLogger
+
+open class ReadingListSchema: Schema {
+    static let DefaultVersion = 1
+
+    public var name: String { return "READINGLIST" }
+    public var version: Int { return ReadingListSchema.DefaultVersion }
+
+    public init() {}
+
+    let itemsTableCreate = "CREATE TABLE IF NOT EXISTS \(TableReadingListItems) (" +
```

Comment:
Would this be a good place to use the new multi-line strings that we have in swift 3?

Reply:
Yes please!